debian/postinst

```diff
@@ -3,14 +3,16 @@
 set -e
 
 . /etc/os-release  # For VERSION_ID
+PREVIOUS_PKG_VER=$2
```

Comment:
`$2` is only the previous package version in the `configure` case, per https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html#summary-of-ways-maintainer-scripts-are-called

Reply:
I agree that is the case, but we don't reference this variable unless we are already down the configure path. What do you suggest we do here instead?  Only set it when we are down configure) path and pass it into is_release_upgrade?

Reply:
We don't _currently_ reference it unless we're down the `configure` path, but we shouldn't make it seem like it will be available in other states.

(You can set it at the start of the configure stanza and it will be set in exactly the same way for the functions that follow it, I believe.)

Reply:
done.